public/css/app.css

```diff
@@ -0,0 +1,5 @@
+@import url(https://fonts.googleapis.com/css?family=Raleway:300,400,600);@charset "UTF-8";/*!
```

Comment:
Maybe .gitignore this to, unless there is any reason to have his in version control?